pkg/operator/controller/garden/add.go

```diff
@@ -27,11 +27,15 @@ func AddToManager(
 	mgr manager.Manager,
 	cfg *config.OperatorConfiguration,
 	identity *gardencorev1beta1.Gardener,
+	gardenClientMap clientmap.ClientMap,
```

Comment:
Strange, here you did it this way (see comment above).

Reply:
This is just a func, but on the comment above, I initialized the `Reconciler` struct with the clientMap and called the struct's `AddToManager`